easybuild/easyconfigs/l/LAMMPS/LAMMPS-20180316-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = "MakeCp"
+
+name = "LAMMPS"
+version = "20180316"
+
+homepage = 'http://lammps.sandia.gov/'
+description = """LAMMPS is a classical molecular dynamics code,
+and an acronym for Large-scale Atomic/Molecular Massively Parallel Simulator."""
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
```

Comment:
@VilleS1 Using a compiler-only toolchain like `GCC` for `LAMMPS` doesn't make much sense?

I suggest removing this easyconfig file.